ksqldb-rest-app/src/test/java/io/confluent/ksql/api/integration/ApiIntegrationTest.java

```diff
@@ -671,6 +802,19 @@ private QueryResponse executeQueryWithVariables(final String sql, final JsonObje
     return new QueryResponse(response.bodyAsString());
   }
 
+  private PrintResponse executePrintQuery(final String sql) {
+    return executePrintQueryWithVariables(sql, new JsonObject());
+  }
+
+  private PrintResponse executePrintQueryWithVariables(final String sql,
```

Comment:
This doesn't look used?

Reply:
I haven't actually touched that. But it is being used by "executePrintQuery", which is again used by the "shouldFailToExecutePrint" test.

Reply:
I see where `executePrintQuery()` is being used but I don't see anywhere that `executePrintQueryWithVariables()` is being used. There's one usage but it looks identical to `executePrintQuery()`. If this understanding is correct, then let's simplify the code to remove `executePrintQueryWithVariables()` and use `executePrintQuery()` everywhere instead.

Reply:
You're right :) I've cleaned the unused code.